packages/code-gen/src/generator/reactQuery/templates/reactQueryFn.tmpl

```diff
@@ -79,6 +79,12 @@ options: UseQueryOptions<{{= responseType }}, AppErrorResponse> = {},
   );
 }
 
+((newline))
+/**
+ * Base key used by {{= funcName }}.queryKey()
+ */
+{{= funcName }}.baseKey = {{= item.uniqueName }};
```

Comment:
We probably want a function here as well, since the base key may be split up in multiple parts later depending on #690, so that we have maximum flexibility

```suggestion
{{= funcName }}.baseKey = function (): QueryKey {
  return [{{= item.uniqueName }}];
}
```